Parsers/ASimWebSession/Parsers/vimWebSessionCitrixNetScaler.yaml

```diff
@@ -0,0 +1,219 @@
+      CommonSecurityLog
+      | where not(disabled)
+      | where (isnull(starttime) or TimeGenerated >= starttime)
+          and (isnull(endtime) or TimeGenerated <= endtime)
+          and (DeviceVendor == "Citrix" and DeviceProduct == "NetScaler")
+      | where DeviceEventClassID == "APPFW" and Activity has_any ("APPFW_STARTURL", "APPFW_XML_cross-site scripting", "APPFW_SAFECOMMERCE", "APPFW_SAFECOMMERCE_XFORM", "APPFW_SIGNATURE_MATCH", "APPFW_XML_ERR_NOT_WELLFORMED", "APPFW_FIELDCONSISTENCY", "APPFW_SQL", "APPFW_BUFFEROVERFLOW_URL", "APPFW_BUFFEROVERFLOW_COOKIE", "APPFW_cross-site scripting", "APPFW_FIELDFORMAT", "APPFW_REFERER_HEADER")
+      | where array_length(httpuseragent_has_any) == 0
+      | where array_length(eventresultdetails_in) == 0
+      | where (array_length(url_has_any) == 0 or RequestURL has_any (url_has_any))
+      | parse-kv AdditionalExtensions as (method: string, geolocation: string, script: string) with (pair_delimiter=";", kv_delimiter="=")
+      | parse RequestURL with * "://" host: string "/" *
+      | extend
+          DeviceAction = trim("[*]+", DeviceAction),
+          temp_SrcMatch = has_any_ipv4_prefix(SourceIP, src_or_any),
+          temp_DstMatch = has_any_ipv4_prefix(RequestURL, ipaddr_has_any_prefix)
+      | lookup EventFieldsLookup on DeviceAction
+      | lookup EventSeverityLookup on DeviceCustomString4
+      | where eventresult == '*' or EventResult =~ eventresult
+      | extend ASimMatchingIpAddr = case(
+                                    array_length(src_or_any) == 0,
+                                    "-",
+                                    temp_SrcMatch and temp_DstMatch,
+                                    "Both",
+                                    temp_SrcMatch,
+                                    "SrcIpAddr",
+                                    temp_DstMatch,
+                                    "DstIpAddr",
+                                    "No match"
+                                )
+      | where ASimMatchingIpAddr != "No match"
+      | extend 
+          Ip_host = iff(host matches regex "(([0-9]{1,3})\\.([0-9]{1,3})\\.([0-9]{1,3})\\.(([0-9]{1,3})))", host, ""),
+          Ip_computer = iff(Computer matches regex "(([0-9]{1,3})\\.([0-9]{1,3})\\.([0-9]{1,3})\\.(([0-9]{1,3})))", Computer, "")
+      | extend
+          host = iff(isempty(Ip_host), host, ""),
+          Computer = iff(isempty(Ip_computer), Computer, ""),
+          AdditionalFields = bag_pack(
+                        "Script", script,
+                        "Event ID", FieldDeviceCustomNumber1,
+                        "HTTP Transaction ID", FieldDeviceCustomNumber2,
+                        "Profile Name", DeviceCustomString1,
+                        "PPE ID", DeviceCustomString2,
+                        "Signature Violation Category", DeviceCustomString6
+                    )
+      | invoke _ASIM_ResolveDvcFQDN('Computer')
+      | invoke _ASIM_ResolveDstFQDN('host')
```

Comment:
I don't think we should do that with 'host' here. This 'host' value seems relevant for HttpHost instead of DstHostname
https://learn.microsoft.com/en-us/azure/sentinel/normalization-schema-web